projects/storefrontlib/src/cms-components/checkout/guards/checkout-auth.guard.ts

```diff
@@ -1,29 +1,38 @@
 import { Injectable } from '@angular/core';
```

Comment:
Can we move this guard to core?

Reply:
I placed it here because it's a guard for the checkout so I placed it with all it's other friends 😉